config/helm/chart/default/templates/Common/csi/clusterrole-csi.yaml

```diff
@@ -1,3 +1,5 @@
+{{- $platformIsSet := printf "%s" (required "Platform needs to be set to kubernetes, openshift" (include "dynatrace-operator.platformSet" .))}}
+{{ if eq (include "dynatrace-operator.needCSI" .) "true" }}
```

Comment:
since we now let the user decide if they want to apply *-csi.yaml or not, do we still need the `needCSI` function?

Reply:
still needed for the `helm` stuff